project/Build.scala

```diff
@@ -17,17 +17,17 @@ import annotation.tailrec
 
 object SumacBuild extends Build {
   
-  lazy val core = Project("core", file("core"), settings = coreSettings).settings(net.virtualvoid.sbt.graph.Plugin.graphSettings: _*)
-  lazy val ext = Project("ext", file("ext"), settings = extSettings).settings(net.virtualvoid.sbt.graph.Plugin.graphSettings: _*) dependsOn(core)
-  lazy val extZk = Project("ext-zk", file("ext-zk"), settings = extZkSettings).settings(net.virtualvoid.sbt.graph.Plugin.graphSettings: _*) dependsOn(core)
+  lazy val core = Project("core", file("core"), settings = coreSettings)
+  lazy val ext = Project("ext", file("ext"), settings = extSettings).dependsOn(core)
+//  lazy val extZk = Project("ext-zk", file("ext-zk"), settings = extZkSettings).dependsOn(core)
```

Comment:
I'm not overly worried about losing this.  I'm not sure anyone still uses it.  If someone speaks up, we can handle it then.

Reply:
I'm all for it - I don't like leaving dead code around either.